.appveyor.yml

```diff
@@ -47,6 +48,9 @@ install:
   - set PATH=%PATH%;%GOPATH%\bin
   - go get -u github.com/golang/dep/cmd/dep
   - dep version
+  - appveyor DownloadFile https://dl.google.com/android/repository/sdk-tools-windows-3859397.zip
+  - 7z x sdk-tools-windows-3859397.zip -o"%ANDROID_HOME%" > nul
+  - yes | C:\android-sdk-windows\tools\bin\sdkmanager.bat --verbose "platform-tools"
```

Comment:
`yes` is not a regular command on Windows. Maybe it's something AppVeyor has installed on top, but that would be new to me.

Reply:
I took the command from an example and it works. But I don't know if it is an AppVeyor feature or what else.